dev-python/logfury/logfury-0.1.2.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Boilerplate library for logging method calls"
+HOMEPAGE="https://github.com/ppolewicz/logfury https://pypi.org/project/logfury/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
```

Comment:
I see `doc` useflag, but it's not used in any way)
Same goes for test, tests are not run.